### ISSUE TITLE
v2.9+ QRouteTab href

### ISSUE THREAD

**Issue by lianee** [quasarframework/quasar#14554]
### What happened?

Edge case issue with the 2.9 release:
UMD version, QRouteTab using only href prop
```
<q-tabs align="left" v-model="tab">
	<q-route-tab href="xxx.html" name="xxx" label="Xxx"></q-route-tab>
</q-tabs>
```

This worked flawlessly with v2.8, but generates error in 2.9

### What did you expect to happen?

no error

### Reproduction URL

https://jsfiddle.net/Liane/qcd8pk9g/

### How to reproduce?

1. click repoduction link
2. see error
3. change quasar loading to @2.8 instead of @2
4. no error

### Flavour

UMD

### Areas

SPA Mode

### Additional context

Adding Vue router and a dummy router fix the issue
`<script src="https://unpkg.com/vue-router@4"></script>`

```
const router = VueRouter.createRouter({
	history: VueRouter.createMemoryHistory(),
	routes: [
		{ path: '', component: {} },
	],
	})
app.use(router)
```

I'm aware that it is mentioned in the docs that "QRouteTab won’t and cannot work with the UMD version if you don’t also install Vue Router." but it seems overkill to install VueRouter when it is not needed, and well, it worked quite nicely until 2.9

**Comment by mobsean** [comment]
error: 
```
TypeError: Cannot read properties of undefined (reading 'fullPath')
```

same as in my issue: #14549

**Comment by rstoenescu** [comment]
Fix will be available in Quasar v2.9.2
